python/tox.ini

```diff
@@ -31,6 +31,7 @@ basepython = python
 deps =
     -r{toxinidir}/requirements_lint.txt
 commands =
+    isort pynessie tests tools
```

Comment:
just wondering:
did we confirm that lint cmd will find invalid import order as well?

is assume its happening because of the added `flake8-isort`, right?
maybe add a comment there similar as we have for `flake8-black`.